bits/expiring_certificate/expiring_certificate.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+from typing import Iterator, List
+
+from octopoes.models import OOI
+from octopoes.models.ooi.certificate import Certificate
+from octopoes.models.ooi.findings import KATFindingType, Finding
+from octopoes.models.ooi.web import Website
+
+THRESHOLD = datetime.timedelta(weeks=2)
```

Comment:
Certificates have a UTC timestamp. We should make sure we check against UTC time and not local timezones.

Reply:
Good point. We're not checking against local time zones, as long as plugins output such dates with a UTC timezone. The plugins related to this are updated to produce date times with UTC time zone.